testing/kuttl/e2e-other/exporter-upgrade/03--check-exporter.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |
+      set -e
+      PRIMARY=$(
+        kubectl get pod --namespace "${NAMESPACE}" \
+          --output name --selector '
+            postgres-operator.crunchydata.com/cluster=exporter,
+            postgres-operator.crunchydata.com/role=master'
+      )
+
+      # Get errors from the exporter
+      ERR=$(kubectl logs --namespace "${NAMESPACE}" "${PRIMARY}" -c exporter | grep -e "Error running query on database")
+      ERR_COUNT=$(echo "$ERR" | wc -l)
+
+      if [[ "$ERR_COUNT" -gt 2 ]]; then
```

Comment:
Why are we checking if the count is greater than 2 here, instead of expecting it to be 0 :question: 

Edit: Oh, I see the Readme; maybe throw a comment here? :pick: